src/main/java/domain/Persons.java

```diff
@@ -73,6 +73,17 @@
     @OneToMany(fetch = FetchType.LAZY, mappedBy = "person")
     private List<Notes> notesList = new ArrayList<>();
 
+
+    @Override
+    public Persons clone(){
+        try{
+            final Persons result = (Persons) super.clone();
+            return result;
+        } catch(final CloneNotSupportedException ex){
+            throw new AssertionError();
```

Comment:
`AssertionError` is not suitable here, throw a `RuntimeException` wrapping `CloneNotSupportedException`